jsk_fetch_robot/jsk_fetch_diagnosis/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(jsk_fetch_diagnosis)
+
+find_package(catkin REQUIRED COMPONENTS
+    roscpp
+    rospy
+    std_msgs
+    message_generation
+)
+
+add_message_files(
+  DIRECTORY msg
+  FILES
+  BoardInfo.msg
+)
+
+add_service_files(
+  DIRECTORY srv
+  FILES
+)
```

Comment:
remove srv lines because there is no `srv` directory.

Reply:
remove add_service_files command